pkg/platform/kube/controller/cronjobpoddeleter.go

```diff
@@ -0,0 +1,69 @@
+package controller
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/nuclio/logger"
+	"k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type cronJobStalePodsDeleter struct {
```

Comment:
```suggestion
type CronJobMonitor struct {
```

one of its usage, is to delete, perhaps later you'd want to do more stuff for other features